verification/curator-service/ui/cypress/e2e/components/All.spec.ts

```diff
@@ -0,0 +1,14 @@
+// import './AcknowledgmentsPage.spec';
```

Comment:
What does this file do?

Reply:
Since Cypress v10 there is now way to run all tests at once locally. By uncommenting those imports and running `All.spec.ts` we get this functionality back. It just makes testing locally easier